### PR TITLE
Deployment config

### PR DESCRIPTION
**Pull request by adamgraves-choices**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of pharmacy-db succeeded (http://pharmacy-db-pr-9.dev.beta.nhschoices.net)

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,5 @@
-sudo: required
```
Comment (st3v3nhunt) [inline .travis.yml:1]: Why remove `sudo: required?` I see that it should provide faster boot times but less memory (which I guess isn't an issue).
https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments

**Review by st3v3nhunt — COMMENTED** [review]
Same question as for nhsuk/profiles-db#50 regarding the removal of `sudo: required` in `.travis.yml`

**Review by st3v3nhunt — APPROVED** [review]
Good for merge from my POV

**Pull request merged**